File: GoChat/src/main/java/com/jw/gochat/base/BaseService.java
package com.jw.gochat.base;

import android.app.Service;

public abstract class BaseService extends Service {

	@Override
	public void onCreate() {
		super.onCreate();
		//((ChatApplication)getApplication()).addService(this);
	}

	@Override
	public void onDestroy() {
		super.onDestroy();
		//((ChatApplication)getApplication()).removeService(this);
	}
}

File: gochatbase/src/main/java/com/jw/gochatbase/base/service/BaseService.kt
package com.jw.gochatbase.base.service

import android.app.Service

abstract class BaseService : Service() {

    override fun onCreate() {
        super.onCreate()
        //((ChatApplication)getApplication()).addService(this);
    }

    override fun onDestroy() {
        super.onDestroy()
        //((ChatApplication)getApplication()).removeService(this);
    }
}